### PR TITLE
[17.0][MIG] event_registration_partner_unique: Migration to version 17.0

### PR DESCRIPTION
**Pull request by juanjosesegui-tecnativa**
@Tecnativa TT49912

Depends on:

- [x] partner_event: https://github.com/OCA/event/pull/382

### CODE REVIEW

**Review by chienandalu — COMMENTED** [review]
The name of the PR isn't correct :sweat_smile:

**Comment by pedrobaeza** [comment]
Please check the red CIs, as the module doesn't even install.

**Comment by juanjosesegui-tecnativa** [comment]
The module depends on partner_event, I suspect it does not install because partner_event is not in version 17.0 of OCA.

**Comment by chienandalu** [comment]
You're suspicions are funded :smile: push the migration of `partner_event` so you can move forward on this one :wink:

**Comment by pedrobaeza** [comment]
/ocabot migration event_registration_partner_unique

If this depends in another PR, you can do: https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference%28s%29-to-another-pull-request%28s%29

**Comment by pedrobaeza** [comment]
And always put in the main comment such dependency:

```
Depends on:

- [ ] partner_event: https://github.com/OCA/event/pull/382
```

**Comment by pedrobaeza** [comment]
You have ignored this part of the guide:

imagen

**Comment by juanjosesegui-tecnativa** [comment]
ping @pedrobaeza @chienandalu

File: event_registration_partner_unique/__manifest__.py [hunk 1]
```diff
@@ -3,11 +3,12 @@
 # Copyright 2018 Tecnativa - Cristina Martin R.
 # Copyright 2020 Tecnativa - Víctor Martínez
 # Copyright 2023 Tecnativa - Carolina Fernandez
+# Copyright 2024 Tecnativa - Juan José Seguí Borja
```
Comment (chienandalu) [inline event_registration_partner_unique/__manifest__.py]: Normally for such small migration changes, better put yourself in the contributors.md file

File: event_registration_partner_unique/views/event_event_view.xml [hunk 2]
```diff
@@ -10,7 +10,7 @@
         <field name="priority" eval="99" />
         <field name="inherit_id" ref="event.view_event_form" />
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='auto_confirm']" position="after">
+            <xpath expr="//field[@name='badge_image']" position="after">
```
Comment (chienandalu) [inline event_registration_partner_unique/views/event_event_view.xml]: It makes more sense after this one
```
            <xpath expr="//field[@name='create_partner']" position="after">
```

**Review by chienandalu — CHANGES_REQUESTED** [review]
Tested :+1: 

Just some small changes:

File: event_registration_partner_unique/views/event_event_view.xml (hunk 2 above)
Comment (juanjosesegui-tecnativa) [reply event_registration_partner_unique/views/event_event_view.xml]: Done!

**Review by chienandalu — COMMENTED** [review]
But put yourself in contributors :sweat_smile:

**Review by chienandalu — APPROVED** [review]
:+1:

**Comment by chienandalu** [comment]
plase review @carolinafernandez-tecnativa

**Comment by carolinafernandez-tecnativa** [comment]
Please Squash administrative commits. Check https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-bots-or-weblate for details.

**Comment by juanjosesegui-tecnativa** [comment]
@carolinafernandez-tecnativa Done!

**Comment by carolinafernandez-tecnativa** [comment]
Please check again I still seing administrative commits and pre-commit auto fixes for migration is missing.

**Comment by carolinafernandez-tecnativa** [comment]
Administrative commits OK

Check please this commit [MIG] event_registration_partner_unique: Migration to 17.0 is empty
and you should not change anything related to migration like this examples in pre-commit https://github.com/OCA/event/pull/390/commits/f38a2b929b798a6cced5178c4530c5f9ceca9c14#diff-858e724775ccf6c429de4636fe90cfabad2b2a15ef432cd347a8eab571e0d21dR10
https://github.com/OCA/event/pull/390/commits/f38a2b929b798a6cced5178c4530c5f9ceca9c14#diff-84d67375c5db3c3d8f88527f5e791e6be3011ae1d5e53927986dce40cfb51545R13

**Comment by carolinafernandez-tecnativa** [comment]
Please Squash administrative commits. There are still some from translation that should be removed. Check https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-bots-or-weblate for details.

**Review by carolinafernandez-tecnativa — APPROVED** [review]
LGTM Thanks :)

**Pull request merged**